packages/core/popper/package.json

```diff
@@ -13,9 +13,11 @@
   "sideEffects": false,
   "scripts": {
     "clean": "rm -rf dist",
-    "prepublish": "yarn clean"
+    "prepublish": "yarn clean",
+    "version": "yarn version"
   },
   "dependencies": {
+    "@babel/runtime-corejs3": "^7.13.10",
```

Comment:
As far as I can see, this is being used by your babel/parcel config prepublish - but not by the peer application.

=> Shouldn't this (now @babel/runtime) be in devDependencies and not dependencies?

Reply:
ah maybe they are needed as per this comment, never mind

> corejs imports are left behind in the ESM bundles

Reply:
It needs to be a dependency because some imports to it may be added, for things like polyfills. So it is indeed a dependency of the package in the end.